markdown/3.0/en/ff-searchbox.md

```diff
@@ -24,58 +24,73 @@ The following code-example shows the aforementioned configuration.
 ```
 For more information, see the [API reference](/api/ff-searchbox#tab=api).
 
-## Extending the input
-The `ff-searchbox` extends a normal `<input>`. Simply add the attribute
-`is="ff-searchbox"` to a standard `<input>` element. On ENTER a search
-query is fired with the current value.
+## Wrapping input element with ff-searchbox
+The `ff-searchbox` wraps a regular HTML `<input>` element. Simply place
+`<input>` between `<ff-searchbox>` tags. On ENTER a search query is fired
+with the current input value.
 ```html
-<input is="ff-searchbox" />
+<ff-searchbox>
+    <input />
+</ff-searchbox>
 ```
 
 ### Placeholder
-Because the `ff-searchbox` is an extension to a normal `<input>` you can
-use all the standard attributes of `<input>` such as `placeholder`.
+Because the `ff-searchbox` does not limit features o HTML it wraps, 
+you can use all the standard attributes of `<input>` such as `placeholder`.
 ```html
-<input is="ff-searchbox" placeholder="Search..."/>
+<ff-searchbox>
+    <input placeholder="Search..." />
+</ff-searchbox>
 ```
 
 ### Select input value on click
 With the `select-onclick` attribute you can define whether the value
 should be selected when the search box gets focus. *(default is false)*
 ```html
-<input is="ff-searchbox" select-onclick="true"/>
+<ff-searchbox select-onclick="true">
+    <input />
+</ff-searchbox>
+
 ```
 
 ## Adding Suggest
 If you want a suggest functionality on your page you can set the
-`use-suggest` attribute to **true** *(default is true)*. You also need
+`use-suggest` attribute to `true` *(default is `true`)*. You also need
```

Comment:
the type used to be String and not Boolean. With the new StringyBoolean I don't know what we should write in the documentation. But it should be consistent with the type in the Api tab (there it is String) and the example below (there it is also String). So probably `"true"` instead of `true` :)

Reply:
I wrapped string values with double quotes - please check now.